client/src/elm/Pages/AcuteIllnessActivity/Utils.elm

```diff
@@ -2565,17 +2570,78 @@ resolveCovid19AcuteIllnessDiagnosis currentDate person isChw measurements =
                                 -- We don't have enough data to make a decision on COVID severity
                                 -- diagnosis, so we report of suspected Covid case.
                                 Just DiagnosisCovid19Suspect
+                    in
+                    case rdtResult of
+                        RapidTestNegative ->
+                            -- On negative result, we can state that this is not a Covid case.
+                            Nothing
+
+                        RapidTestIndeterminate ->
+                            -- This is not an option at Covid RDT, so we can
+                            -- rule it out.
+                            Nothing
+
+                        RapidTestPositiveAndPregnant ->
+                            -- Covid with pregnancy always concidered as severe case.
+                            Just DiagnosisSevereCovid19
+
+                        RapidTestUnableToRun ->
+                            -- We treat Unable to run as if test was positive.
```

Comment:
`We treat Unable to run` seems wrong, please fix comment.

Reply:
@nedSaf can you explain what seems wrong please?

Reply:
@anvmn It's not clear what you mean by the comment, so I can't tell you exactly because I didn’t understand the comment, just make it clearer please.

Reply:
@nedSaf done